internal/pkg/config/pbkdf2.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package config
+
+import (
+	"bytes"
+	"errors"
+)
+
+type PBKDF2 struct {
+	Iterations int `config:"iterations"`
+	KeyLength  int `config:"key_length"`
+	SaltLength int `config:"salt_length"`
+
+	// BlockSize must be a factor of aes.BlockSize
+	BlockSize int `config:"block_size"`
```

Comment:
```suggestion
```
is this actually used somewhere?

Reply:
I copied the configuration from Elastic Agent. I will remove that as that is not used here.

Reply:
Removed